syntax = "proto3";

option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.trade.response";
option java_outer_classname = "UnLockTradeResponseProto";
message UnLockTradeResponse {
	bool success = 1; //false-失败，true-成功
}